client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -414,6 +418,12 @@ public static ContentType createContentType(final XContentType xContentType) {
         return ContentType.create(xContentType.mediaTypeWithoutParameters(), (Charset) null);
     }
 
+    static Request indicesExist(IndicesExistsRequest request) {
+        String endpoint = endpoint(request.indices(), Strings.EMPTY_ARRAY, "");
+
+        return new Request(HttpHead.METHOD_NAME, endpoint, emptyMap(), null);
```

Comment:
we should add support here for some missing parameters, see https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/indices.exists.json .

Reply:
@javanna I implemented this. Two values I could extract from IndicesExistsRequest I did. Please double check, since I have the feeling I missed something.